code/modules/projectiles/projectile.dm

```diff
@@ -124,18 +126,19 @@
 	uncross_scheduled = null
 	original_target_turf = null
 	starting_turf = null
+	hit_atoms = list()
```

Comment:
Also maybe just hit_atoms = null instead of creating a new list that just gets deleted right after anyways.